### ISSUE TITLE
Modify repo to have only one branch for Backdrop contrib

### ISSUE THREAD

**Issue by izmeez** [rudy880719/domain#3]
The repo will need to be modified so that it only contains one branch as described in the docs, https://docs.backdropcms.org/converting-modules-from-drupal

**Comment by rudy880719** [comment]
correct, but I have left the drupal version for review differences, I would like stpaultim, argiepiano and yorkshire-pudding to check what I did

**Comment by rudy880719** [comment]
They started the port first, even in the backdrop repository there is already a domain accesses created by them, we must wait for their decision.

**Comment by izmeez** [comment]
@rudy880719 Have you looked at the issues in the other port, https://github.com/backdrop-contrib/domain/issues or tested it to see how the two compare?

**Comment by izmeez** [comment]
@rudy880719 are you on the backdrop zulip chat? https://backdrop.zulipchat.com/login/?next=/%23narrow%2Fstream%2F218635-Backdrop
It may be a good place to discuss the steps moving forward. Or possibly in https://forum.backdropcms.org

**Comment by rudy880719** [comment]
@izmeez Hello, I have not seen the differences between the two branches, I have not been able to look at that, I could only test their port and some things did not work, that is why I started my port from scratch, yes, I just installed the zulip chat, maybe you could help me ask and see the possible scenarios. I wanted to ask you if you have tried all the modules of my port.

**Comment by izmeez** [comment]
@rudy880719 I have not tested all the modules and features, I have only just started to test the module. It is an extensive and complex module that offers many options that may fit many different use cases. I was thinking it may be helpful to discuss this in the zulip chat, although you may have some other specific questions. I can start a thread for the discussion.
